scripts/create_tables.sql

```diff
@@ -1 +1,27 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE patients(
+id SERIAL PRIMARY KEY,
+surname TEXT,
+given_name TEXT,
+gender TEXT,
+born_on TEXT,
+insurance TEXT,
+);
+
+CREATE TABLE doctors(
+id SERIAL PRIMARY KEY,
+surname TEXT,
+given_name TEXT,
+gender TEXT,
+born_on TEXT,
+specialty TEXT,
+);
+
+CREATE TABLE appointments(
+id SERIAL PRIMARY KEY,
+date TEXT,
+time TEXT,
+comment TEXT,
+doctor_id
+patient_id
+);
```

Comment:
Not quite. Every column needs to have a datatype, and the `doctor_id` and `patient_id` on the `appointments` table need to reference those existing tables.

```sql
CREATE TABLE appointments(
  date TEXT, time TEXT, comment TEXT,
  doctor_id INTEGER REFERENCES doctors(id),
  patient_id INTEGER REFERENCES patients(id)
);
```